Sources/Datadog/Core/Utils/ValuePublisher.swift

```diff
@@ -48,7 +48,7 @@ internal class ValuePublisher<Value> {
         }
     }
 
-    init(initialValue: Value) {
+    required init(initialValue: Value) {
```

Comment:
This is necessary to use this init in protocol extension with `Self` requirement, such as `RandomMock`. This could also be fixed by turning `ValuePublisher` into a `final class`.

Reply:
It could be `final` indeed, but it's OK for now I guess 👍

Reply:
I kind of prefer `final` as it is stricter.